src/Service/HaalCentraalToStufBGService.php

```diff
@@ -135,6 +135,31 @@ public function getAllRelatives(Source $source, array $ingeschrevenPersoon): ?ar
     }//end getAllRelatives()
 
 
+    /**
+     * Checks if the nationality of the ingeschreven persoon is Dutch.
+     *
+     * @param array $ingeschrevenPersoon Already fetched ingeschrevenPersoon to check the nationality.
+     *
+     * @return array|null If the nationality of a ingeschreven persoon is Dutch.
+     */
+    public function checkDutchNationality(array $ingeschrevenPersoon): ?array
+    {
+        // Set the nationality default on false.
+        $dutchNationality = "false";
+        foreach ($ingeschrevenPersoon['nationaliteiten'] as $nationality) {
+            // If the omschrijving is 'Nederlandse' or the code is '0001' then set the bool to true.
+            if ($nationality['nationaliteit']['omschrijving'] === 'Nederlandse'
+                || $nationality['nationaliteit']['code'] === '0001'
```

Comment:
Did you check if this code will always be 0001? 
(this is the case, probably, but if we are unsure only checking Nederlandse might be better)

Reply:
This is what I could find about the code, maybe we also have to use the 0002? https://brp-api.github.io/Haal-Centraal-BRP-bevragen/features/test-report-persoon-nationaliteit-gba.html

<img width="514" alt="image" src="https://github.com/CommonGateway/HaalCentraalToStufBG/assets/55533449/375eada5-011c-465c-8baa-c91083aa5a76">

Reply:
Check, 👍